Unable to login in Control Panel

### Bug description

After updating to Statamic 6 I'm not able to login to the control panel anymore. 

On the login screen, below the Passkey button I get a validation message `Authentication failed.` In the console I see the following message: 

```
AbortError: Authentication ceremony was sent an abort signal:
WebAuthnError https://mywebsite.nl/vendor/statamic/cp/build/assets/index-CyJXg3fB.js:264
...
```

Tried all the obvious actions like clearing the cache etc. But no luck unfortunately.

### How to reproduce

Update from 5.0 to 6.0

### Logs

```shell
nothing here
```

### Environment

```yaml
Environment
Application Name: Digital Deer
Laravel Version: 12.49.0
PHP Version: 8.4.17
Composer Version: 2.4.3
Environment: production
Debug Mode: OFF
URL: mywebsite.nl
Maintenance Mode: OFF
Timezone: Europe/Amsterdam
Locale: nl

Cache
Config: NOT CACHED
Events: NOT CACHED
Routes: NOT CACHED
Views: CACHED

Drivers
Broadcasting: log
Cache: file
Database: sqlite
Logs: daily
Mail: smtp
Queue: sync
Session: file

Storage
public/storage: NOT LINKED

Statamic
Addons: 1
Sites: 1
Stache Watcher: Disabled
Static Caching: full
Version: 6.1.0 PRO

Statamic Addons
aryehraber/statamic-captcha: 1.16.0
```

### Installation

Fresh statamic/statamic site via CLI

### Additional details

_No response_

I am running into the same issue I think.

I am able to login with my email and password without any issues but when I try the "Passkey" button I also get the "Authentication failed." error.

<img width="1929" height="1197" alt="Image" src="https://github.com/user-attachments/assets/55ed00a6-2213-4162-9e48-fdb733db2ed2" />

It says this in the javascript logs:

```
AbortError: Authentication ceremony was sent an abort signal
    WebAuthnError https://www.llresearch.test/vendor/statamic/cp/build/assets/index-CyJXg3fB.js:264
    identifyAuthenticationError https://www.llresearch.test/vendor/statamic/cp/build/assets/index-CyJXg3fB.js:265
    startAuthentication https://www.llresearch.test/vendor/statamic/cp/build/assets/index-CyJXg3fB.js:265
Caused by: AbortError: Cancelling existing WebAuthn API call for new one
    createNewAbortSignal https://www.llresearch.test/vendor/statamic/cp/build/assets/index-CyJXg3fB.js:264
    startAuthentication https://www.llresearch.test/vendor/statamic/cp/build/assets/index-CyJXg3fB.js:265
index-CyJXg3fB.js:265:3686
```

When you click the passkey button it does a request to `/cp/auth/passkeys/options` which gets a 200 Response

```json
{
    "challenge":"lkqit8wa3aYDLbKvJ9V6vyvhKD3FEhFxYSlNQicYLNU",
    "allowCredentials":[],
    "userVerification":"required"
}
```

But it still says that it failed?

<img width="1929" height="1197" alt="Image" src="https://github.com/user-attachments/assets/8f9d9733-f359-4899-8a46-80308b2dc0ba" />

But I was able to create the passkey first on the Control Panel without any issues.

<img width="1382" height="984" alt="Image" src="https://github.com/user-attachments/assets/df30886f-2712-4307-856f-8bdce8835402" />

I also ran `php please auth:migration` and the webauthn tables are in the database... but my users are set to the `file` driver, not `eloquent`. I checked the config files for changes in Statamic v6 and I see it is set to `eloquent` as the default now?

But I can't seem to import the users with the eloquent-driver to test it, I'm not sure if that's related or a separate issue:

```
php please eloquent:import-users

   TypeError 

  class_parents(): Argument #1 ($object_or_class) must be of type object|string, null given

  at vendor/laravel/framework/src/Illuminate/Support/helpers.php:108
    104▕         }
    105▕ 
    106▕         $results = [];
    107▕ 
  ➜ 108▕         foreach (array_reverse(class_parents($class) ?: []) + [$class => $class] as $class) {
    109▕             $results += trait_uses_recursive($class);
    110▕         }
    111▕ 
    112▕         return array_unique($results);

      +16 vendor frames 
```

--- a/resources/js/composables/passkey.js
+++ b/resources/js/composables/passkey.js
@@ -1,5 +1,5 @@
 import { ref } from 'vue';
-import { startAuthentication, browserSupportsWebAuthn } from '@simplewebauthn/browser';
+import { startAuthentication, browserSupportsWebAuthn, WebAuthnAbortService } from '@simplewebauthn/browser';
 import axios from 'axios';
 
 export function usePasskey() {
@@ -19,6 +19,7 @@ export function usePasskey() {
             try {
                 startAuthResponse = await startAuthentication({ optionsJSON, useBrowserAutofill });
             } catch (e) {
+                if (e.name === 'AbortError') return;
                 console.error(e);
                 error.value = __('Authentication failed.');
                 if (!useBrowserAutofill) waiting.value = false;
@@ -37,6 +38,10 @@ export function usePasskey() {
         }
     }
 
+    function cancel() {
+        WebAuthnAbortService.cancelCeremony();
+    }
+
     function handleError(e) {
         if (e.response) {
             const { message } = e.response.data;
@@ -52,5 +57,6 @@ export function usePasskey() {
         waiting,
         supported,
         authenticate,
+        cancel,
     };
 }

--- a/resources/js/composables/passkey.js
+++ b/resources/js/composables/passkey.js
@@ -19,7 +19,7 @@ export function usePasskey() {
             try {
                 startAuthResponse = await startAuthentication({ optionsJSON, useBrowserAutofill });
             } catch (e) {
-                if (e.name === 'AbortError') return;
+                if (e.name === 'AbortError' || e.name === 'NotAllowedError') return;
                 console.error(e);
                 error.value = __('Authentication failed.');
                 if (!useBrowserAutofill) waiting.value = false;

--- a/resources/js/pages/auth/Login.vue
+++ b/resources/js/pages/auth/Login.vue
@@ -3,7 +3,7 @@ import Head from '@/pages/layout/Head.vue';
 import Outside from '@/pages/layout/Outside.vue';
 import { AuthCard, Input, Field, Button, Separator, Checkbox, ErrorMessage } from '@ui';
 import { Link, router } from '@inertiajs/vue3';
-import { computed, onMounted, ref, watch } from 'vue';
+import { computed, onMounted, onUnmounted, ref, watch } from 'vue';
 import { usePasskey } from '@/composables/passkey';
 
 defineOptions({ layout: Outside });
@@ -86,6 +86,8 @@ async function loginWithPasskey(useBrowserAutofill = false) {
 onMounted(() => {
     if (showPasskeyLogin.value) loginWithPasskey(true);
 });
+
+onUnmounted(() => passkey.cancel());
 </script>
 
 <template>
@@ -133,7 +135,7 @@ onMounted(() => {
                             :icon="passkey.waiting.value ? null : 'key'"
                             :disabled="passkey.waiting.value"
                             :loading="passkey.waiting.value"
-                            @click="loginWithPasskey"
+                            @click="loginWithPasskey()"
                         />
                         <ErrorMessage v-if="passkey.error.value" :text="passkey.error.value" />
                     </template>

